apps/ui/src/views/SpaceUser.vue

```diff
@@ -0,0 +1,199 @@
+<script setup lang="ts">
+import { addressValidator as isValidAddress } from '@/helpers/validation';
+import {
+  _vp,
+  autoLinkText,
+  getCacheHash,
+  getSocialNetworksLink,
+  shortenAddress
+} from '@/helpers/utils';
+import { getNetwork, supportsNullCurrent } from '@/networks';
+import type { Space, UserActivity } from '@/types';
+import type { VotingPower, VotingPowerStatus } from '@/networks/types';
+
+const props = defineProps<{ space: Space }>();
+
+const route = useRoute();
+const usersStore = useUsersStore();
+const spacesStore = useSpacesStore();
+const { param } = useRouteParser('id');
+const { resolved, address, networkId } = useResolve(param);
+const { setTitle } = useTitle();
+const { getCurrent } = useMetaStore();
+
+const userActivity = ref<UserActivity>({ vote_count: 0, proposal_count: 0 } as UserActivity);
+const loaded = ref(false);
+const votingPowers = ref([] as VotingPower[]);
+const votingPowerStatus = ref<VotingPowerStatus>('loading');
+
+const network = computed(() => getNetwork(props.space.network));
+
+const userId = computed(() => route.params.user as string);
+
+const user = computed(() => usersStore.getUser(userId.value));
+
+const socials = computed(() => getSocialNetworksLink(user.value));
+
+const shareMsg = computed(() => encodeURIComponent(window.location.href));
+
+const cb = computed(() => getCacheHash(user.value?.avatar));
+
+const formattedVotingPower = computed(() => {
+  const votingPower = votingPowers.value.reduce((acc, b) => acc + b.value, 0n);
+  const decimals = Math.max(...votingPowers.value.map(votingPower => votingPower.decimals), 0);
+
+  const value = _vp(Number(votingPower) / 10 ** decimals);
+
+  if (props.space.voting_power_symbol) {
+    return `${value} ${props.space.voting_power_symbol}`;
+  }
+
+  return value;
+});
+
+const navigation = computed(() => [
+  { label: 'Statement', route: 'space-user-statement' },
+  { label: 'Delegators', route: 'space-user-delegators' },
+  { label: 'Proposals', route: 'space-user-proposals', count: userActivity.value?.proposal_count },
+  { label: 'Votes', route: 'space-user-votes', count: userActivity.value?.vote_count }
```

Comment:
On design, it is "LATEST VOTES"
<img width="456" alt="Untitled 6" src="https://github.com/user-attachments/assets/71be185b-8006-4e41-8a2e-48bde0c61026">